packages/components/src/components/tabs/tabs.scss

```diff
@@ -86,6 +87,18 @@
         &--is-hidden {
             display: none;
         }
+        &--is-scrollable-and-active {
+            position: relative;
+
+            &:after {
+                border-bottom: 2px solid var(--brand-red-coral);
+                content: '';
```

Comment:
What is the purpose of `content: ''` here?

Reply:
https://www.w3.org/TR/CSS22/generate.html#content